TrackCat/settings.py

```diff
@@ -56,6 +56,8 @@
 	'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
 	'django.contrib.messages.middleware.MessageMiddleware',
 	'django.middleware.clickjacking.XFrameOptionsMiddleware',
+	'django.contrib.auth.middleware.AuthenticationMiddleware',
+	'django.contrib.messages.middleware.MessageMiddleware',
```

Comment:
You have these 2 midllewares already in this list. You don't need them twice (so just remove them).